recipe/meta.yaml

```diff
@@ -25,25 +25,31 @@ requirements:
     - python >=3.6
     # Required
     - affine >=2.2
+    - blas =*.*=mkl
     - click >=7.0
     - cmocean >=2.0
     - dask >=1.2
     - fiona >=1.8
-    - gdal >=2.4,<3.0
-    - matplotlib-base >=3.0
+    - gdal >=2.4
```

Comment:
We had issues with gdal 3+. I cannot remember what that was unfortunately. Are you sure that thsi works?

Reply:
According to git blame, Alicja removed the "<3.0" restriction in this commit: https://github.com/dcs4cop/xcube/commit/c1175beebb97432361aac1458b259508475ebbfc , and it has been gone from `environment.yml` ever since. After half a year I would guess that we can consider it safe, but I can re-add the restriction if you think it's too risky. (In that case we should consider also re-adding it to `environment.yml`.)

Reply:
That's fine, please specify 
```
- gdal >=3.0
```